handler/src/main/java/io/netty/handler/stream/ChunkedStream.java

```diff
@@ -99,8 +100,14 @@ public void close() throws Exception {
         in.close();
     }
 
+    @Deprecated
```

Comment:
Maybe it is just an IDE (eclipse) thing that it doesn't show as deprecated on the inheriting type ... I'll have to look a this more ... does IDEA correctly propagate the deprecated status?